testsuite/MDAnalysisTests/analysis/test_distances.py

```diff
@@ -29,8 +29,7 @@
 
 import MDAnalysis.analysis.distances
 
-from numpy.testing import (assert_equal, assert_array_equal, assert_almost_equal, 
-                           assert_array_almost_equal,assert_allclose)
+from numpy.testing import (assert_equal ,assert_allclose)
```

Comment:
pep8 space and I don't think parentheses are needed
```suggestion
from numpy.testing import assert_equal, assert_allclose
```